web/libs/ui/src/tailwind.config.js

```diff
@@ -2,15 +2,7 @@ import tokens from "./tokens/tokens";
 
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [
-    "./apps/**/*.{js,jsx,ts,tsx}",
-    "./libs/app-common/src/**/*.{js,jsx,ts,tsx}",
-    "./libs/core/src/**/*.{js,jsx,ts,tsx}",
-    "./libs/editor/src/**/*.{js,jsx,ts,tsx}",
-    "./libs/datamanager/src/**/*.{js,jsx,ts,tsx}",
-    "./libs/ui/src/**/*.{js,jsx,ts,tsx}",
-    "./libs/storybook/**/*.{js,jsx,ts,tsx}",
-  ],
+  content: ["./apps/**/*.{js,jsx,ts,tsx}", "./libs/**/*.{js,jsx,ts,tsx}"],
```

Comment:
discussing with @bmartel - if the e2e/integration test environments are getting timed out its likely because of picking up too many things here. please take another look

Reply:
solution maybe to just revert to what we had before to make sure we dont get extra/unexpected stuff